ui/src/views/Tasks/TaskGroup/index.jsx

```diff
@@ -164,16 +169,98 @@ export default class TaskGroup extends Component {
     searchTerm: null,
   };
 
+  static unsubscribe() {
+    if (!TaskGroup.listener) {
+      return;
+    }
+
+    TaskGroup.listener.unsubscribe();
+    TaskGroup.listener = null;
```

Comment:
Maybe the same question as about the module-level globals, but why are these class-level properties, and static methods?